Description: Add keywords key to .desktop files
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>
--- a/savers/cosmos-slideshow-ukui.desktop.in.in
+++ b/savers/cosmos-slideshow-ukui.desktop.in.in
@@ -7,4 +7,5 @@
 Terminal=false
 Type=Application
 Categories=Screensaver;
+Keywords=UKUI;screensaver;slideshow;cosmos;
 OnlyShowIn=UKUI;
--- a/savers/footlogo-floaters-ukui.desktop.in.in
+++ b/savers/footlogo-floaters-ukui.desktop.in.in
@@ -7,4 +7,5 @@
 Terminal=false
 Type=Application
 Categories=Screensaver;
+Keywords=UKUI;screensaver;logo;bubbles;
 OnlyShowIn=UKUI;
--- a/savers/gnomelogo-floaters-ukui.desktop.in.in
+++ b/savers/gnomelogo-floaters-ukui.desktop.in.in
@@ -7,4 +7,5 @@
 Terminal=false
 Type=Application
 Categories=Screensaver;
+Keywords=UKUI;screensaver;GNOME;logo;bubbles;
 OnlyShowIn=UKUI;
--- a/savers/personal-slideshow-ukui.desktop.in.in
+++ b/savers/personal-slideshow-ukui.desktop.in.in
@@ -7,4 +7,5 @@
 Terminal=false
 Type=Application
 Categories=Screensaver;
+Keywords=UKUI;screensaver;custom;slideshow;pictures;folder;
 OnlyShowIn=UKUI;
--- a/savers/popsquares-ukui.desktop.in.in
+++ b/savers/popsquares-ukui.desktop.in.in
@@ -7,4 +7,5 @@
 Terminal=false
 Type=Application
 Categories=Screensaver;
+Keywords=UKUI;screensaver;grid;pulsing;colors;
 OnlyShowIn=UKUI;
--- a/data/ukui-screensaver-preferences.desktop.in
+++ b/data/ukui-screensaver-preferences.desktop.in
@@ -6,6 +6,7 @@
 Terminal=false
 Type=Application
 Categories=GTK;Settings;DesktopSettings;
+Keywords=UKUI;screensaver;preferences;
 OnlyShowIn=UKUI;
 StartupNotify=true
 X-UKUI-Bugzilla-Bugzilla=UKUI
--- a/src/ukui-screensaver.desktop.in.in
+++ b/src/ukui-screensaver.desktop.in.in
@@ -6,6 +6,7 @@
 Exec=ukui-screensaver
 OnlyShowIn=UKUI;
 NoDisplay=true
+Keywords=UKUI;screensaver;locker;
 X-UKUI-Autostart-Phase=Application
 X-UKUI-Autostart-Notify=true
 X-UKUI-Bugzilla-Bugzilla=UKUI
